scripts/IssueMetrics/performanceclassifier.py

```diff
@@ -9,6 +9,7 @@
 from github import Github, Repository
 import string
 
+nltk.download('punkt')
```

Comment:
I add this line because when I run in docker I have this error that ask me to download `punkt`
![Screen Shot 2021-02-11 at 11 15 26 PM](https://user-images.githubusercontent.com/35275868/107788974-3fa9e280-6d0e-11eb-9bb0-a1960b020cba.png)